composeApp/src/commonMain/kotlin/com/overdrive/cruiser/views/MySpotsView.kt

```diff
@@ -31,170 +31,213 @@ import androidx.compose.runtime.setValue
+                imageVector = Icons.Filled.Share,
+                contentDescription = "Share",
+                tint = Color.Black
+            )
+        }
 
+        Column(modifier = Modifier.padding(16.dp)) {
+            SpotOnField(modifier = Modifier.fillMaxWidth()) {
+                Row(
+                    modifier = Modifier.padding(16.dp),
+                    verticalAlignment = Alignment.CenterVertically
+                ) {
+                    Text(spot.address)
+                }
+            }
+            SpotOnField(modifier = Modifier.fillMaxWidth()) {
+                Row(
+                    modifier = Modifier.padding(16.dp),
+                    verticalAlignment = Alignment.CenterVertically
+                ) {
+                    Text("$5/hour")
+                    Spacer(modifier = Modifier.weight(1f))
+                    Text("$40/day")
                 }
-                Spacer(modifier = Modifier.weight(1f))
-
-                Button(
-                    onClick = onAddSpotClick,
-                    shape = CircleShape,
-                    colors = ButtonDefaults.buttonColors(backgroundColor = Color.LightGray),
-                    modifier = Modifier
-                        .align(Alignment.End)
-                        .clip(CircleShape)
-                        .size(64.dp)
+            }
+            SpotOnField(modifier = Modifier.fillMaxWidth()) {
+                Row(
+                    modifier = Modifier.padding(16.dp),
+                    verticalAlignment = Alignment.CenterVertically
                 ) {
-                    Text("+")
+                    Text("226 total bookings")
                 }
             }
+            Button(
+                onClick = { onAddSpotClick() },
+                shape = RoundedCornerShape(12.dp),
+                colors = ButtonDefaults.buttonColors(backgroundColor = Color(0xFFF9784B)),
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(55.dp)
+                    .clip(RoundedCornerShape(12.dp))
+            ) {
+                Text(
+                    text = "Edit Spot",
+                    color = Color.White
+                )
+            }
+
+            Spacer(modifier = Modifier.weight(1f))
+
+            Button(
+                onClick = {
+                    scope.launch {
+                        showDialog = true
+                    }
+                },
+                shape = RoundedCornerShape(12.dp),
+                colors = ButtonDefaults.buttonColors(
+                    backgroundColor = Color.White,
+                ),
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(55.dp)
+                    .clip(RoundedCornerShape(12.dp))
+                    .border(2.dp, Color(0xFFF9784B), RoundedCornerShape(12.dp))
+            ) {
+                Text(
```

Comment:
There is an update problem with deleted spots remaining in the UI. I don't think we are gonna demo this so probably fine to leave this.

Reply:
This is an issue with spots being booked, if they're in the booking table they fail to delete. We'll probably need to figure out some logic for this later but might have to leave this for now.

Reply:
Yeah okay, what we actually want is more of an "Archive" function (so that our records are still valid), but yeah that's a server side thing so this is fine for now.